### ISSUE TITLE
Merenisco, MI should be Central Time

### ISSUE THREAD

**Issue by ironwallaby** [darkskyapp/tz-lookup#25]
https://en.wikipedia.org/wiki/Marenisco_Township,_Michigan

**Comment by ironwallaby** [comment]
Confirmed the bug:
```
  1) tzlookup
       should return "America/Chicago" given 46.3661, -89.5969:
     Error: expected "America/Detroit" to equal "America/Chicago"
      at Context.<anonymous> (test.js:29:17)
```

**Comment by ironwallaby** [comment]
This is fixed v6.1.10.

Turns out this should be `America/Menominee`, not `America/Chicago`.
